platform/pulpcore/tasking/services/scheduler.py

```diff
@@ -15,6 +15,11 @@
 from pulpcore.tasking.services import worker_watcher
 
 
+# The import below is not used in this module, but it needs to be kept here. This module is the
+# first and only Pulp module to be imported by celerybeat, and by importing pulpcore.app.logs, it
+# configures the celerybeat logging to log as Pulp does.
+import pulpcore.app.logs  # noqa
```

Comment:
This got removed on 3.0-dev. Perhaps it is only here because of a rebase mishap?